app/Services/Eggs/Sharing/EggImporterService.php

```diff
@@ -45,4 +58,17 @@ public function handle(UploadedFile $file): Egg
             return $egg;
         });
     }
+
+    protected function recursiveStringReplace(array $data, array $replacements): array
```

Comment:
Wouldn't it make sense to place this logic in a trait (or a different class / service)?

Reply:
This was removed, and replaced with lances suggestion.